index.js

```diff
@@ -86,15 +87,39 @@ class Sqlite extends Base {
           console.error('ERR_DB_SQLITE', err)
         })
         this._maybeRunSqlAtStart(next)
+      },
+      next => {
+        this.db.allAsync = promisify(this.db.all.bind(this.db))
+        this.db.getAsync = promisify(this.db.get.bind(this.db))
+        this.db.runAsync = this.runAsync.bind(this)
```

Comment:
runAsync is not needed here since you do it below

Reply:
I don't understand... Here, I'm only exposing it on the internal sqlite db connection directly. Sounds like that's a good idea?

Reply:
oh sorry I didn't notice this.db, I think that's not a good practice, we should do `this.allAsync = promisify(this.db.all.bind(this.db))`, we don't want to do prototype pollution in a dep